test/e2e/common/pods.go

```diff
@@ -26,7 +26,7 @@ import (
 
 	"golang.org/x/net/websocket"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I still see this change in the diff. But the comment as resolved. Has there been any other conversation where this was approved?

Reply:
I will remove it, it's auto imported by the ide